extensions/src/AWSSDK.Extensions.CloudFront.Signers/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("AWSSDK.Extensions.Cloudfront.Signers")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Amazon.com, Inc")]
+[assembly: AssemblyProduct("AWS SDK for .NET extensions for Amazon CloudFront")]
+[assembly: AssemblyDescription("AWS SDK for .NET extensions for Amazon CloudFront")]
+[assembly: AssemblyCopyright("Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.")]
+[assembly: AssemblyTrademark("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+#if NETFRAMEWORK
+[assembly: AssemblyVersion("4.0")]
```

Comment:
Is there a reason why netframework is being set to a different version number?

Reply:
Yes, see this in the `Other noteworthy changes` section of the V4 tracker:

> In .NET Standard 2.0, .NET Core 3.1 and .NET 8 the AssemblyVersion will match the AssemblyFileVersion. In .NET Framework 4.7.2 target AssemblyVersion will continue to use the V3 pattern of locking the version to the first 2 parts of the the AssemblyFileVersion. This is done because .NET Framework treats the AssemblyVersion as part of the identity assembly. This causes assembly collisions and forced recompilation when mixing versions of the SDK.